app/src/hooks/useCardImageHeight.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from "react";
+
+export function useCardImageHeight(cardRef) {
```

Comment:
Eventually, this hook needs a set of tests.

Reply:
Actually, none of the hooks in DC have tests.

For now, can you add jsdoc style comments
```
/**
 * ....
 */
```